packages/cli/src/commands/storybook.js

```diff
@@ -56,6 +68,7 @@ export const handler = ({ open, port, build, buildDirectory }) => {
       !build && `--port ${port}`,
       !build && '--no-version-updates',
       !build && `--static-dir "${staticAssetsFolder}"`,
+      noManagerCache && '--no-manager-cache',
```

Comment:
- continued from first suggestion
```suggestion
      !managerCache && '--no-manager-cache',
```